train_forward.py

```diff
@@ -81,11 +81,14 @@ def create_gta_features(model: Tacotron,
 
     if force_gta:
         print('Creating Ground Truth Aligned Dataset...\n')
```

Comment:
Similar to the comment in the train_tacotron.py, I would suggest to move force_gta functionality out of this sctipt or at least not to mix within the same if-else conctruct with training options

Reply:
Same as above, lets do that in a separate refactoring.